lxqt-session/src/sessionapplication.cpp

```diff
@@ -88,7 +82,10 @@ bool SessionApplication::startup()
 
     loadEnvironmentSettings(settings);
     // loadFontSettings(settings);
-    if(QX11Info::isPlatformX11()) {
+
+    LXQt::Platform::PLATFORM platform = LXQt::Platform::getPlatform();
+    if(platform == LXQt::Platform::X11) {
+        // X11 session is started
```

Comment:
This is equivalent to:
``` cpp
    if (QGuiApplication::platformName() == QLatin1String("xcb")) {
``` 
My question is why is https://github.com/lxqt/liblxqt/pull/286 needed ?
I might be looking at the tree and not the forest.

Reply:
> My question is why is lxqt/liblxqt#286 needed ?

I have exactly the same question. We know that Qt does it in a standard way. Wouldn't we complicate our code unnecessarily?

Reply:
@selairi 
I think wrapping is usually done where there are multiple back-ends. We use Qt to know about platforms and, as far as I know, it does its job well.

As for `menu-cache`, it's removed from lxqt-panel by default. If it's active in lxqt-runner, we'll need to do a similar thing there (after 0.17). Also see https://github.com/lxqt/libfm-qt/issues/417.

Thanks for making all LXQt components work under Wayland!